### DISCUSSION TITLE
[Java] Publish Java Jars to Maven Central or JCenter

### DISCUSSION THREAD

**Discussion by mrussek** [microsoft/Recognizers-Text#1797] [Ideas]
**Is your feature request related to a problem? Please describe.**
Most widely used Java projects publish artifacts to widely used Java repositories such as Maven Central or JCenter. This allows developers to quickly download and use these artifacts. It appears (and I may be mistaken) that there are no currently published artifacts on Maven Central, making it more difficult than necessary to get started.

**Describe the solution you'd like**
Add the maven deploy plugin to the `pom.xml` file for each module, and integrate it with the current CI pipeline.

**Describe alternatives you've considered**
My organization currently has a source fork of your repository with a custom CI pipeline. For most Java Libraries we have automation around Maven Central.

**Comment by tellarin** [comment]
@mrussek, thanks for the feedback. The current Java recognizers don't provide the same coverage as other platforms yet, but we'll consider publishing alpha packages to Maven.
Would you be willing to contribute a PR with the package creation/validation process that the project can leverage?
